index.html

```diff
@@ -2,43 +2,90 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css"/>
 </head>
 <body>
-    <header>
-        <a class="logo" href="#">My Logo</a>
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        <meta charset="UTF-8">
+        <title>Fork Me! FCC: Test Suite Template</title>
+        <link rel="stylesheet" href="css/style.css">
```

Comment:
- [ ] Your CSS file has not been uploaded yet so please make sure you upload your CSS file in order to show off your page design.

Reply:
- [ ] **[OPTIONAL]** Good job updating your previous suggestion **as the previous reviewer requested!** I have a small optional suggestion for you: **instead of creating a new comment, you can simply edit the previous ones.** This way, it **will be even easier to keep track of the changes.** Keep up the good work! 😉

  #### You can edit your review comments here:
  ![image](https://user-images.githubusercontent.com/107952724/235543841-6988b256-c4cb-4d52-b4d3-fd54c6d2adfc.png)